### PR TITLE
x11-themes/experience: EAPI bump 0 -> 7

### PR DESCRIPTION
**Pull request by Moiman**
Package-Manager: Portage-2.3.36, Repoman-2.3.9

### CODE REVIEW

File: x11-themes/experience/Manifest [hunk 1]
```diff
@@ -1 +1 @@
-DIST experience-gtk-3.04.tar.gz 108140 BLAKE2B 0909eb0a55fcc27f806ccebcf4ebfa8fef53e753227bf04accd4460c0ea4507d9627c7a2796a4fb6fad1841b9c032de0397e72ce62a623a0cd6756578340ddce SHA512 528c282fae43da76d88401dc1225cb8ce357e3c2786d5848bfdc3560692939837a813d7689f2136ef1cf8f832390a8619c4736c0ff99ea5a2c1818be2cd47a81
+DIST GTK2-EXperience.tar.gz 108140 BLAKE2B 0909eb0a55fcc27f806ccebcf4ebfa8fef53e753227bf04accd4460c0ea4507d9627c7a2796a4fb6fad1841b9c032de0397e72ce62a623a0cd6756578340ddce SHA512 528c282fae43da76d88401dc1225cb8ce357e3c2786d5848bfdc3560692939837a813d7689f2136ef1cf8f832390a8619c4736c0ff99ea5a2c1818be2cd47a81
```
Comment (johu) [inline x11-themes/experience/Manifest]: Please fix the SRC_URI by using the "->" operator to have a versioned distfile.

File: x11-themes/experience/experience-3.04.ebuild [hunk 2]
```diff
@@ -4,9 +4,8 @@
 EAPI=0
 
 DESCRIPTION="GTK+2 themes which copy and improve the look of XP Luna"
-HOMEPAGE="https://art.gnome.org/themes/gtk2/1058"
-#SRC_URI="https://art.gnome.org/download/themes/gtk2/1058/GTK2-EXperience.tar.gz"
-SRC_URI="http://freshmeat.net/redir/${PN}/50795/url_tgz/${PN}-gtk-${PV}.tar.gz"
+HOMEPAGE="https://web.archive.org/web/20130730053042/https://art.gnome.org/themes/gtk2/1058"
+SRC_URI="mirror://gnome/teams/art.gnome.org/themes/gtk2/GTK2-EXperience.tar.gz"
```
Comment (johu) [inline x11-themes/experience/experience-3.04.ebuild]: same as in in -r1

Comment (Moiman) [reply x11-themes/experience/experience-3.04.ebuild]: "->" isn't allowed in eapi=0

Comment (johu) [reply x11-themes/experience/experience-3.04.ebuild]: On the other hand the unversioned distfile is a QA issue as well. So how about to leave the SRC_URI for the in -r0 untouched?

**Comment by Moiman** [comment]
> What's the point of EAPI bump here ? Is there a plan to get rid of older EAPIs ?

https://wiki.gentoo.org/wiki/Project:Quality_Assurance/Current_projects
> Migrating ebuilds from EAPIs 1 and 2 (deprecated) to newer EAPIs, as well as packages using EAPI 0 which are not system packages. Priority should go to any packages which are still being regularly updated but with the outdated EAPI.

**Comment by johu** [comment]
@EvaSDK Do you know repoman?

**Comment by johu** [comment]
Thank you. (I have reverted the -r0 SRC_URI change because of unversioned distfile).

**Pull request closed without merging**